SpatialGDK/Source/SpatialGDK/Private/Utils/OpUtils.cpp

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Utils/OpUtils.h"
```

Comment:
Can you alphabetize these

Reply:
primary header first though, then alphabetize no?

Reply:
Whoops my bad.